src/main/java/io/rainfall/ScenarioRun.java

```diff
@@ -143,9 +143,13 @@ public ScenarioRun assertion(final Assertion actual, final Assertion expected) {
 
       timer.scheduleAtFixedRate(stats, reportIntervalMillis, reportIntervalMillis);
 
+      System.out.println("Starting scenario.");
```

Comment:
it would be good to use the logger (info?) for this so it will let users decide if they want to display it or not - and by the same occasion, changing other existing System.print.out to logger